tests/x11/rstudio_server.pm

```diff
@@ -62,14 +62,20 @@ sub run() {
     # log out at last and close firefox
     assert_and_click("rstudio_server-sign-out");
     assert_screen("rstudio_server-login-username");
+
     send_key('alt-f4');
 
-    # optionally click away the close all tabs window
-    assert_screen([qw(rstudio_server-firefox_quit-and-close-tabs generic-desktop)]);
+    check_screen([qw(rstudio_server-firefox_quit-and-close-tabs generic-desktop)], timeout => 60);
     if (match_has_tag('rstudio_server-firefox_quit-and-close-tabs')) {
+        # optionally click away the close all tabs window => FF should close
         click_lastmatch();
-        assert_screen('generic-desktop');
+    } elsif (!match_has_tag('generic-desktop')) {
+        # firefox is stuck and just greys out but does not display the "close
+        # all tabs" dialog => kill it
+        record_soft_failure('Firefox froze for 60s and does not want to close');
```

Comment:
have you seen https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13130/checks?check_run_id=3461849409#step:6:1769 ? soft-fails must reference valid product issues. If you don't have a bug report for that use

```suggestion
        record_info('Workaround', 'Firefox froze for 60s and does not want to close');
```

Reply:
Thanks! I've applied this fix, but I must confess that I wasn't able to figure out what the test suite was complaining about from the output.